inference/server/main.py

```diff
@@ -106,6 +110,70 @@ def maybe_add_debug_api_keys():
         raise
 
 
+@app.get("/auth/login/discord")
+async def login_discord(request: Request):
+    domain = str(request.url).split("/auth/")[0]
```

Comment:
the domain part should probably come from the settings, not from the request, otherwise there might be some nasty security exploits possible

Reply:
Good spot, changed this to a setting